lib/gtranslate.py

```diff
@@ -132,35 +138,39 @@ def translate(text, dest_lang, src_lang=None):
         src_lang: the text's language, or autodetect if it's None
 
     Returns:
-        translated text if all went well or None otherwise
-        source language if all went well or None otherwise
+        Translation(NamedTuple):
+            msg: translated text
+            lang: the detected language
```

Comment:
C'est mieux de décrire 
```suggestion
        Translation or None: A Translation object if the translation succeeded, otherwise None is returned.
```